### PR TITLE
Adjustments regarding the new state of ocrd-main branch

### PR DESCRIPTION
**Pull request by markusweigelt**
- Add OCR-D Manager workflows as volume
- Adjust paths and scripts to support newer state of `ocrd-main` branch with OCR profiles

### CODE REVIEW

File: _resources/kitodo/before_startup.sh [hunk 1]
```diff
@@ -25,3 +25,11 @@ if [ -n "$OCRD_MANAGER" ]; then
 
   ssh-keyscan -H -p ${OCRD_MANAGER_PORT:-22} $OCRD_MANAGER_HOST,$OCRD_MANAGER_IP >>/etc/ssh/ssh_known_hosts
 fi
+
+# link OCR-D Manager workflow into OCR profiles folder
+if [ ! -d "/tmp/kitodo/overwrites/data/ocr/profiles" ]; then
+  mkdir -p /tmp/kitodo/overwrites/data/ocr/profiles/
+  mkdir -p /usr/local/ocrd-manager/workflows/
+
+  ln -s /usr/local/ocrd-manager/workflows /tmp/kitodo/overwrites/data/ocr/profiles/workflows
```
Comment (bertsky) [inline _resources/kitodo/before_startup.sh:34]: So why not use `/tmp/kitodo/overwrites/data/ocr/profiles/workflows` as mount point directly?

**Review by bertsky — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -141,8 +139,11 @@ $(CONTROLLER_MODELS)/ocrd-resources/ocrd-tesserocr-recognize/frak2021.traineddat
 	wget -O $@ https://ub-backup.bib.uni-mannheim.de/~stweil/tesstrain/frak2021/tessdata_best/frak2021-0.905.traineddata
 
 build:
+# do not use cache for kitodo-app service, otherwise the latest version of ocrd-main branch will not be used
+	docker compose build --no-cache kitodo-app
```
[the remark this answers is not in the document]
Comment (markusweigelt) [reply Makefile:143]: Great hint!

File: _resources/kitodo/before_startup.sh (hunk 1 above)
Comment (markusweigelt) [reply _resources/kitodo/before_startup.sh:34]: However, if the workflows in OCR-D Manager changed afterwards or another workflow is created, they will not be available in Kitodo.Production. `/tmp/kitodo/overwrites/data/` is copies to `/usr/local/kitodo/` and a volume is mounted to latter directory. In order to represent the dynamic of later workflow management in monitor, I had decided for this form of the implementation. In a productive environment you would mount the workflow folder from the manager direct.

Comment (bertsky) [reply _resources/kitodo/before_startup.sh:34]: Oh, great, thanks for the explanation!

**Pull request merged**